textattack/datasets/huggingface_nlp_dataset.py

```diff
@@ -0,0 +1,55 @@
+from textattack.datasets import TextAttackDataset
+from textattack.shared import TokenizedText
+
+import nlp
+import random
+
+def get_nlp_dataset_columns(dataset):
```

Comment:
Am I missing something? Why are all the outputs "labels"? Do we supporot seq2seq datasets?